examples/block_entity.js

```diff
@@ -2,14 +2,18 @@
  * This example demonstrates how easy it is to create a bot
  * that fetches monster spawners mob type
  */
-const mineflayer = require('mineflayer')
+import { createBot } from 'mineflayer'
 
 if (process.argv.length < 4 || process.argv.length > 6) {
-  console.log('Usage : node block_entity.js <host> <port> [<name>] [<password>]')
+  console.log('Usage : node blcok_entity.js <host> <port> [<name>] [<password>]')
   process.exit(1)
 }
 
-const bot = mineflayer.createBot({
+// TODO: commands
+console.log('Commands :\n' +
+  '  spawner in message')
```

Comment:
Why console log? It's not intended to be used as a bot. It's intended to act as an example of a bot. Anyone who use that file will look at the code before running it.